backend/app/member_handlers_test.go

```diff
@@ -0,0 +1,100 @@
+package app
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/codescalersinternships/Flyspray/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateNewMember(t *testing.T) {
+	app, err := NewApp("./test.db")
+	assert.NoError(t, err, "failed to connect to database")
+	app.setRoutes()
+	testCases := []struct {
+		name           string
+		member         models.Member
+		expectedStatus int
+	}{
+		{
+			name:           "creating valid member returns status 201",
+			member:         models.Member{UserID: 1, ProjectID: 2},
+			expectedStatus: http.StatusCreated,
+		},
+		{
+			name:           "creating invalid member returns status 500",
+			member:         models.Member{},
```

Comment:
not sure this is the best way to structure the tests as we need some way to verify that the resources are created successfully, either alter the struct to verify the data is correct or consider not using table based tests